app/javascript/articles/PodcastArticle.jsx

```diff
@@ -0,0 +1,34 @@
+import { h } from 'preact';
+import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
+
+export const PodcastArticle = ({ article }) => {
```

Comment:
This as pulled from https://github.com/thepracticaldev/dev.to/blob/master/app/assets/javascripts/utilities/buildArticleHTML.js.erb#L3 to create the Preact component.